aws/ec2/deploy/home/ec2-user/snappydata/fetch-distribution.sh

```diff
@@ -41,29 +41,30 @@ getLatestUrl() {
 SNAPPY_HOME_DIR="snappydata-${SNAPPYDATA_VERSION}-bin"
 SNAPPY_HOME_DIR_NO_BIN="snappydata-${SNAPPYDATA_VERSION}"
 
-if [[ "${SNAPPYDATA_VERSION}" = "LATEST" ]]; then
-  getLatestUrl
-  extract
-elif [[ ! -d ${SNAPPY_HOME_DIR} ]] && [[ ! -d ${SNAPPY_HOME_DIR_NO_BIN} ]]; then
-  wget -q https://github.com/SnappyDataInc/snappydata/releases
-  URL_PART=`grep -o "/SnappyDataInc/snappydata/releases/download/[a-zA-Z0-9.\/\-]**${SNAPPYDATA_VERSION}-bin.tar.gz" releases`
-  GREP_RESULT=`echo $?`
-  if [[ ${GREP_RESULT} != 0 ]]; then
-    # Try without '-bin'
-    URL_PART=`grep -o "/SnappyDataInc/snappydata/releases/download/[a-zA-Z0-9.\/\-]**${SNAPPYDATA_VERSION}.tar.gz" releases`
-    GREP_RESULT=`echo $?`
-  fi
-  if [[ ${GREP_RESULT} != 0 ]]; then
-    echo "Did not find binaries for ${SNAPPYDATA_VERSION}, instead will use the latest version."
-    getLatestUrl
-  else
-    URL="https://github.com${URL_PART}"
-  fi
-  extract
-else
-  if [[ -d ${SNAPPY_HOME_DIR_NO_BIN} ]]; then
-    SNAPPY_HOME_DIR=${SNAPPY_HOME_DIR_NO_BIN}
-  fi
-fi
-
+#if [[ "${SNAPPYDATA_VERSION}" = "LATEST" ]]; then
```

Comment:
I think we should just keep this code uncommented. It'll be needed when we start supporting multiple versions.